meta-resin-common/recipes-support/resin-logs/resin-logs/resin-logs

```diff
@@ -4,12 +4,26 @@ set -e
 
 . /usr/sbin/resin-vars
 
-if [ "$PERSISTENT_LOGGING" == "true" ]; then
-    echo "[INFO] Configure persistent logs in /mnt/state/root-overlay/var/log/journal"
-    echo -e "[Journal]\nStorage=persistent\n" > /etc/systemd/journald.conf.d/resin-logs.conf
-elif [ "$PERSISTENT_LOGGING" == "false" ]; then
-    echo "[INFO] Configure volatile logs in /run/log/journal"
-    echo -e "[Journal]\nStorage=volatile\n" > /etc/systemd/journald.conf.d/resin-logs.conf
+machineID=`cat /etc/machine-id`
+
+if [ "$PERSISTENT_LOGGING" == "true" ] && [ ! -d /var/log/journal ]; then
+    systemctl start var-log-journal.mount
+    # Send flush signal to journald
+    killall -USR1 systemd-journald
+elif [ -d /var/log/journal ]; then
```

Comment:
Can it not enter this elif in this case?

[ "$PERSISTENT_LOGGING" == "true" ] && [ -d /var/log/journal ]